reveal/run.js

```diff
@@ -73,6 +74,37 @@
     });
   }
 
+  function hasRemoteQueryParameter() {
+    return window.location.search.match(/[?&]remote[&]?/i);
+  }
+
+  function isRemoteMode() {
+    return Reveal.isOverview()
+                  ? false
+                  : hasRemoteQueryParameter();
+  }
+
+  function enableRemoteMode() {
+    if (window.location.search.match(/[?]/))
+      window.location.search = window.location.search + '&remote';
+    else
+      window.location.search = '?remote';
```

Comment:
Could you add some braces there and lines 103-106? It's fine to omit braces when condition and consequence are on the same line but here it might trip the next contributor up.